mod/search/views/default/resources/search/index.php

```diff
@@ -61,6 +61,27 @@
 	$order = 'desc';
 }
 
+// Get object containers: container of comments
+$container_guids = array();
+if (!empty($container_guid)) {
+	$container_guids[] = (int)$container_guid; // Add group container to the container's list
+	$comment_subtype_id = get_subtype_id('object', 'comment');
```

Comment:
what about discussion replies?

Reply:
You are right, I forgot discussion replies, I will fix it.